src/uwtools/cli.py

```diff
@@ -693,9 +707,9 @@ def _check_file_vs_format(file_arg: str, format_arg: str, args: Args) -> Args:
     return args
 
 
-def _check_quiet_vs_verbose(args) -> Args:
-    if args.get(STR.quiet) and args.get(STR.verbose):
-        _abort("Specify at most one of %s, %s" % (_switch(STR.quiet), _switch(STR.verbose)))
+def _check_verbosity(args) -> Args:
+    if args.get(STR.quiet) and (args.get(STR.debug) or args.get(STR.verbose)):
+        _abort("--quiet may not be used with --debug or --verbose")
```

Comment:
I appreciate this updated message, but it would still be a good idea to use `_switch(STR.quiet)` etc. in place of the literal versions, to keep strings DRY.

Reply:
Oh I missed that one, thanks!